cmd/bosun/sched/sched.go

```diff
@@ -43,6 +43,7 @@ type Schedule struct {
 	LastCheck     time.Time
 	nc            chan interface{}
 	notifications map[*conf.Notification][]*State
+	lastLogTimes  map[expr.AlertKey]time.Time
```

Comment:
Let's move it to State. It's just a few extra bytes, and I don't want to start spreading data around in maps when we already have a place per alertkey.